internal/pkg/mining/worker.go

```diff
@@ -72,8 +71,7 @@ type MessageSource interface {
 
 // A MessageApplier processes all the messages in a message pool.
 type MessageApplier interface {
-	// ApplyMessagesAndPayRewards applies all state transitions related to a set of messages.
-	ApplyMessagesAndPayRewards(ctx context.Context, st state.Tree, vms vm.StorageMap, messages []*types.UnsignedMessage, minerOwnerAddr address.Address, bh *types.BlockHeight, ancestors []block.TipSet) ([]*consensus.ApplyMessageResult, error)
+	// Dragons: add something back or remove
```

Comment:
To be consistent with your other Dragons, this should be replaced with an interface that asks for new messages and gets back valid messages (by running this on the implementation)

Reply:
its test code that will get nuked on the next PR once the legacy vmcontext goes away.